static/css/diff.less

```diff
@@ -1,3 +1,5 @@
+@import (less) "components/backtoTop.less";
```

Comment:
You only need to import this in one place - all-js.css.

Right now this is only loading styles on certain places.
On home page it appears unstyled for example.